src/pluto/exception.cr

```diff
@@ -1,8 +1,11 @@
 class Pluto::Exception < ::Exception
-  getter error_code : LibJPEGTurbo::ErrorCode
+  getter error_code : Int32
+
+  def initialize(@error_code : Int32)
+  end
```

Comment:
Should this have a default error message at least to indicate what went wrong? i.e. `"Error #{@error_code} received during image file I/O"`